Vagrantfile

```diff
@@ -0,0 +1,96 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
```

Comment:
It would be nice to add a quick comment to your Readme to specify that running the app in a VM is an option, and provide some guidance for what that relies on & how the user can start that